recipe/0102-Fix-error-using-gcc-8.patch

```diff
@@ -0,0 +1,44 @@
+From f876e817fa98ab7f14bf498c3a96e3a7a29ec823 Mon Sep 17 00:00:00 2001
+From: Nishidha Panpaliya <npanpa23@in.ibm.com>
+Date: Thu, 29 Jul 2021 09:40:52 -0400
+Subject: [PATCH] Fix error using gcc 8
+
+---
+ dali/operators/audio/mfcc/mfcc.h | 10 +++++-----
+ 1 file changed, 5 insertions(+), 5 deletions(-)
+
+diff --git a/dali/operators/audio/mfcc/mfcc.h b/dali/operators/audio/mfcc/mfcc.h
+index c6d4d06e..eb55f03c 100644
+--- a/dali/operators/audio/mfcc/mfcc.h
++++ b/dali/operators/audio/mfcc/mfcc.h
+@@ -88,22 +88,22 @@ class MFCC : public Operator<Backend> {
+   void GetArguments(const workspace_t<Backend> &ws) {
+     auto nsamples = ws.template InputRef<Backend>(0).shape().size();
+     DctArgs arg;
+-    arg.ndct = spec_.template GetArgument<int>("n_mfcc");
```

Comment:
Why not make a PR directly to DALI?

Reply:
This is just a back port of: https://github.com/NVIDIA/DALI/pull/1320

Reply:
Well, relooking It does look like we have that fix in a few other places.
Gcc8 also seems to choke on this block, which may need the same:
https://github.com/NVIDIA/DALI/blob/v1.3.0/dali/operators/image/remap/warp.h#L317-L321

Reply:
We'll PR upsteam if this all works.

Reply:
Sure, I'll raise the PR DALI too.

Reply:
The change is similar to NVIDIA/DALI#1320, but is about mfcc.h.
If this works we would be happy to include it in the mainline.